src/models/fixtures/city.js

```diff
@@ -1,16 +1,11 @@
 import fixture from 'can-fixture';
+import City from '../city';
 
 const store = fixture.store([
   { state: 'CA', name: 'Casadina' },
   { state: 'NT', name: 'Alberny' }
-],{});
+], City.connection.algebra);
```

Comment:
I'm not. It seems redundant to me since the `algebra` is passed to `superMap`. This is what I landed on:

```
const algebra = new set.Algebra(
  set.props.id('name')
);

City.connection = superMap({
  url: baseUrl + '/api/cities',
  Map: City,
  List: City.List,
  name: 'city',
  algebra
});
```